src/puppetlabs/puppetdb/queue.clj

```diff
@@ -250,21 +250,32 @@
                  file-extension))))
 
 (defn cmdref->cmd [q cmdref]
+  "Returns the command associated with cmdref, or logically false if
+  the file is missing (i.e. someone has deleted it)."
```

Comment:
Hmm, well the way I tend to think if it, at least for an api spec is that there's no way to put the paste back in the tube once you've been more specific (callers would be well within their rights to use `nil?` if the function says it returns `nil`), but it doesn't matter here - happy to change it if you prefer.